code/datums/supply_packs.dm

```diff
@@ -911,6 +923,7 @@
 	category = "Civilian Department"
 	contains = list(/obj/item/chem_grenade/cleaner = 4,
 					/obj/item/spraybottle/cleaner = 2,
+					/obj/item/reagent_containers/glass/bottle/cleaner = 2,
```

Comment:
this is not mentioned in your PR description

Reply:
"Removed the porta-brig from the Sec crate, made it it's own unique purchase, added sec belt, helmet and vest to said crate. **Also added 2 bottle of space cleaner to Janitor Resupply while I was at it.**"

First paragraph.